src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1792,7 +1792,7 @@ internal bool CanSaveAs(object parameters)
         private void InternalSaveAs(string path, SaveContext saveContext, bool isBackup = false)
         {
             try
-            {
+            {                
```

Comment:
extra spaces

Reply:
@filipeotero Can you address this as well before we merge?